2023/09/20 14:04:27 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "srbac/authitem/install". in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
HTTP_REFERER=http://localhost/yii-2/
---
2023/09/20 14:04:37 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "srbac". in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php?r=srbac
HTTP_REFERER=http://localhost/yii-2/
---
2023/09/20 14:07:24 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "srbac". in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php?r=srbac
HTTP_REFERER=http://localhost/yii-2/
---
2023/09/20 14:11:26 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid.
in C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php (34)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (51)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 14:11:26 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid in C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php(34): CDbCommand->queryAll()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CWebUser.php(817): SDbAuthManager->checkAccess('Authority', 'admin', Array)
#3 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(51): CWebUser->checkAccess('Authority')
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(306): AuthitemController->beforeAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('install')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
HTTP_REFERER=http://localhost/yii-2/
---
2023/09/20 14:20:32 [error] [exception.CException] CException: CDbAuthManager.connectionID "connectionString" is invalid. Please make sure it refers to the ID of a CDbConnection application component. in C:\xampp\htdocs\yii-2\framework\web\auth\CDbAuthManager.php:592
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CDbAuthManager.php(59): CDbAuthManager->getDbConnection()
#1 C:\xampp\htdocs\yii-2\framework\base\CModule.php(394): CDbAuthManager->init()
#2 C:\xampp\htdocs\yii-2\framework\base\CModule.php(103): CModule->getComponent('authManager')
#3 C:\xampp\htdocs\yii-2\protected\modules\srbac\SrbacModule.php(244): CModule->__get('authManager')
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(43): SrbacModule->isInstalled()
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(306): AuthitemController->beforeAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('install')
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#10 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
HTTP_REFERER=http://localhost/yii-2/
---
2023/09/20 14:21:52 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid.
in C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php (34)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (51)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 14:21:52 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid in C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php(34): CDbCommand->queryAll()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CWebUser.php(817): SDbAuthManager->checkAccess('Authority', 'admin', Array)
#3 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(51): CWebUser->checkAccess('Authority')
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(306): AuthitemController->beforeAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('install')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
HTTP_REFERER=http://localhost/yii-2/
---
2023/09/20 14:22:49 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid.
in C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php (34)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (51)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 14:22:49 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid in C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php(34): CDbCommand->queryAll()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CWebUser.php(817): SDbAuthManager->checkAccess('Authority', 'admin', Array)
#3 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(51): CWebUser->checkAccess('Authority')
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(306): AuthitemController->beforeAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('install')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
HTTP_REFERER=http://localhost/yii-2/
---
2023/09/20 14:27:26 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid.
in C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php (34)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (51)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 14:27:26 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid in C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php(34): CDbCommand->queryAll()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CWebUser.php(817): SDbAuthManager->checkAccess('Authority', 'admin', Array)
#3 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(51): CWebUser->checkAccess('Authority')
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(306): AuthitemController->beforeAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('install')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
HTTP_REFERER=http://localhost/yii-2/
---
2023/09/20 14:42:24 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid.
in C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php (34)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (51)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 14:42:24 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid in C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php(34): CDbCommand->queryAll()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CWebUser.php(817): SDbAuthManager->checkAccess('Authority', 'admin', Array)
#3 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(51): CWebUser->checkAccess('Authority')
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(306): AuthitemController->beforeAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('install')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
---
2023/09/20 14:45:43 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid.
in C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php (34)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (51)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 14:45:43 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid in C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php(34): CDbCommand->queryAll()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CWebUser.php(817): SDbAuthManager->checkAccess('Authority', 'admin', Array)
#3 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(51): CWebUser->checkAccess('Authority')
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(306): AuthitemController->beforeAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('install')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
---
2023/09/20 14:51:55 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid.
in C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php (34)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (51)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 14:51:55 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid in C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php(34): CDbCommand->queryAll()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CWebUser.php(817): SDbAuthManager->checkAccess('Authority', 'admin', Array)
#3 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(51): CWebUser->checkAccess('Authority')
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(306): AuthitemController->beforeAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('install')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
---
2023/09/20 14:51:56 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid.
in C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php (34)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (51)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 14:51:56 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid in C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php(34): CDbCommand->queryAll()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CWebUser.php(817): SDbAuthManager->checkAccess('Authority', 'admin', Array)
#3 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(51): CWebUser->checkAccess('Authority')
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(306): AuthitemController->beforeAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('install')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
---
2023/09/20 14:51:57 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid.
in C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php (34)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (51)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 14:51:57 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid in C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php(34): CDbCommand->queryAll()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CWebUser.php(817): SDbAuthManager->checkAccess('Authority', 'admin', Array)
#3 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(51): CWebUser->checkAccess('Authority')
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(306): AuthitemController->beforeAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('install')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
---
2023/09/20 14:51:57 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid.
in C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php (34)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (51)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 14:51:57 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid in C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php(34): CDbCommand->queryAll()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CWebUser.php(817): SDbAuthManager->checkAccess('Authority', 'admin', Array)
#3 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(51): CWebUser->checkAccess('Authority')
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(306): AuthitemController->beforeAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('install')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
---
2023/09/20 14:51:57 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid.
in C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php (34)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (51)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 14:51:57 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii-test.items' doesn't exist. The SQL statement executed was: SELECT name, type, description, t1.bizrule, t1.data, t2.bizrule AS bizrule2, t2.data AS data2 FROM items t1, assignments t2 WHERE name=itemname AND userid=:userid in C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\yii-2\protected\modules\srbac\components\SDbAuthManager.php(34): CDbCommand->queryAll()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CWebUser.php(817): SDbAuthManager->checkAccess('Authority', 'admin', Array)
#3 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(51): CWebUser->checkAccess('Authority')
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(306): AuthitemController->beforeAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('install')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('srbac/authitem/...')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
---
2023/09/20 14:58:01 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 15:03:26 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 15:04:26 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 15:04:26 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 15:04:27 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 15:04:27 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 15:08:37 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 15:54:46 [error] [exception.CException] CException: CCaptchaValidator.action "captcha" is invalid. Unable to find such an action in the current controller. in C:\xampp\htdocs\yii-2\framework\validators\CCaptchaValidator.php:77
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\validators\CCaptchaValidator.php(49): CCaptchaValidator->getCaptchaAction()
#1 C:\xampp\htdocs\yii-2\framework\validators\CValidator.php(201): CCaptchaValidator->validateAttribute(Object(UserAdmin), 'verifyCode')
#2 C:\xampp\htdocs\yii-2\framework\base\CModel.php(159): CValidator->validate(Object(UserAdmin), Array)
#3 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(810): CModel->validate(NULL)
#4 C:\xampp\htdocs\yii-2\protected\controllers\UserAdminController.php(93): CActiveRecord->save()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): UserAdminController->actionCreate()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin/creat...')
#16 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin/create
HTTP_REFERER=http://localhost/yii-2/index.php?r=userAdmin/create
---
2023/09/20 15:56:54 [error] [exception.CException] CException: CCaptchaValidator.action "captcha" is invalid. Unable to find such an action in the current controller. in C:\xampp\htdocs\yii-2\framework\validators\CCaptchaValidator.php:77
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\validators\CCaptchaValidator.php(49): CCaptchaValidator->getCaptchaAction()
#1 C:\xampp\htdocs\yii-2\framework\validators\CValidator.php(201): CCaptchaValidator->validateAttribute(Object(UserAdmin), 'verifyCode')
#2 C:\xampp\htdocs\yii-2\framework\base\CModel.php(159): CValidator->validate(Object(UserAdmin), Array)
#3 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(810): CModel->validate(NULL)
#4 C:\xampp\htdocs\yii-2\protected\controllers\UserAdminController.php(93): CActiveRecord->save()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): UserAdminController->actionCreate()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin/creat...')
#16 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin/create
HTTP_REFERER=http://localhost/yii-2/index.php?r=userAdmin/create
---
2023/09/20 16:00:29 [error] [exception.CException] CException: CCaptchaValidator.action "captcha" is invalid. Unable to find such an action in the current controller. in C:\xampp\htdocs\yii-2\framework\validators\CCaptchaValidator.php:77
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\validators\CCaptchaValidator.php(49): CCaptchaValidator->getCaptchaAction()
#1 C:\xampp\htdocs\yii-2\framework\validators\CValidator.php(201): CCaptchaValidator->validateAttribute(Object(UserAdmin), 'verifyCode')
#2 C:\xampp\htdocs\yii-2\framework\base\CModel.php(159): CValidator->validate(Object(UserAdmin), Array)
#3 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(810): CModel->validate(NULL)
#4 C:\xampp\htdocs\yii-2\protected\controllers\UserAdminController.php(93): CActiveRecord->save()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): UserAdminController->actionCreate()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin/creat...')
#16 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin/create
HTTP_REFERER=http://localhost/yii-2/index.php?r=userAdmin/create
---
2023/09/20 16:00:57 [error] [exception.CException] CException: CCaptchaValidator.action "captcha" is invalid. Unable to find such an action in the current controller. in C:\xampp\htdocs\yii-2\framework\validators\CCaptchaValidator.php:77
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\validators\CCaptchaValidator.php(49): CCaptchaValidator->getCaptchaAction()
#1 C:\xampp\htdocs\yii-2\framework\validators\CValidator.php(201): CCaptchaValidator->validateAttribute(Object(UserAdmin), 'verifyCode')
#2 C:\xampp\htdocs\yii-2\framework\base\CModel.php(159): CValidator->validate(Object(UserAdmin), Array)
#3 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(810): CModel->validate(NULL)
#4 C:\xampp\htdocs\yii-2\protected\controllers\UserAdminController.php(93): CActiveRecord->save()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): UserAdminController->actionCreate()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin/creat...')
#16 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin/create
HTTP_REFERER=http://localhost/yii-2/index.php?r=userAdmin/create
---
2023/09/20 16:04:16 [error] [exception.CException] CException: Property "UserIdentity._id" is not defined. in C:\xampp\htdocs\yii-2\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii-2\protected\components\UserIdentity.php(39): CComponent->__set('_id', '4')
#1 C:\xampp\htdocs\yii-2\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii-2\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\htdocs\yii-2\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\htdocs\yii-2\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#5 C:\xampp\htdocs\yii-2\protected\controllers\SiteController.php(94): CModel->validate()
#6 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii-2/index.php?r=site/login
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:04:55 [error] [exception.CException] CException: Property "UserIdentity.id_user" is not defined. in C:\xampp\htdocs\yii-2\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii-2\protected\components\UserIdentity.php(39): CComponent->__set('id_user', '4')
#1 C:\xampp\htdocs\yii-2\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii-2\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\htdocs\yii-2\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\htdocs\yii-2\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#5 C:\xampp\htdocs\yii-2\protected\controllers\SiteController.php(94): CModel->validate()
#6 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii-2/index.php?r=site/login
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:05:23 [error] [exception.CException] CException: Property "UserIdentity._id" is not defined. in C:\xampp\htdocs\yii-2\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii-2\protected\components\UserIdentity.php(39): CComponent->__set('_id', '4')
#1 C:\xampp\htdocs\yii-2\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii-2\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\htdocs\yii-2\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\htdocs\yii-2\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#5 C:\xampp\htdocs\yii-2\protected\controllers\SiteController.php(94): CModel->validate()
#6 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii-2/index.php?r=site/login
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:05:28 [error] [exception.CException] CException: Property "UserIdentity._id" is not defined. in C:\xampp\htdocs\yii-2\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii-2\protected\components\UserIdentity.php(39): CComponent->__set('_id', '4')
#1 C:\xampp\htdocs\yii-2\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii-2\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\htdocs\yii-2\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\htdocs\yii-2\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#5 C:\xampp\htdocs\yii-2\protected\controllers\SiteController.php(94): CModel->validate()
#6 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii-2/index.php?r=site/login
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:09:45 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('view')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin/view')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin/view&id=4
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:11:32 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('view')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin/view')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin/view&id=4
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:13:47 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin/')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin/
---
2023/09/20 16:13:50 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin/')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin/
---
2023/09/20 16:21:40 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "admin". in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('admin')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php?r=admin
---
2023/09/20 16:21:56 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin/')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin/
---
2023/09/20 16:22:13 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "userAdmin". in C:\xampp\htdocs\yii-2\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CController.php(270): CController->missingAction('userAdmin')
#1 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('userAdmin')
#2 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('site/userAdmin')
#3 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii-2/index.php?r=site/userAdmin
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:30:42 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:39:03 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin
---
2023/09/20 16:39:10 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:39:44 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:44:58 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:49:26 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:51:23 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 16:52:11 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin')
#9 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 17:00:22 [error] [exception.CException] CException: CWebUser and its behaviors do not have a method or closure named "getLevel". in C:\xampp\htdocs\yii-2\framework\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\yii-2\protected\controllers\SiteController.php(88): CComponent->__call('getLevel', Array)
#1 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('login')
#6 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#7 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#8 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/yii-2/index.php?r=site/login
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 17:01:27 [error] [exception.CException] CException: CWebUser and its behaviors do not have a method or closure named "getLevel". in C:\xampp\htdocs\yii-2\framework\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\yii-2\protected\controllers\SiteController.php(91): CComponent->__call('getLevel', Array)
#1 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('login')
#6 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#7 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#8 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/yii-2/index.php?r=site/login
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 17:08:24 [error] [php] Undefined variable: userAdmin (C:\xampp\htdocs\yii-2\framework\base\CComponent.php(614) : eval()'d code:1)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\base\CComponent.php(614): eval()
#1 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(393): CAccessRule->evaluateExpression()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(289): CAccessRule->isExpressionMatched()
#3 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(138): CAccessRule->isUserAllowed()
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter()
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CAccessControlFilter->filter()
#6 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): UserAdminController->filterAccessControl()
#7 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#9 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): UserAdminController->runActionWithFilters()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): UserAdminController->run()
#11 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=userAdmin/admin
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 17:08:57 [error] [php] Undefined variable: userAdmin (C:\xampp\htdocs\yii-2\framework\base\CComponent.php(614) : eval()'d code:1)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\base\CComponent.php(614): eval()
#1 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(393): CAccessRule->evaluateExpression()
#2 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(289): CAccessRule->isExpressionMatched()
#3 C:\xampp\htdocs\yii-2\framework\web\auth\CAccessControlFilter.php(138): CAccessRule->isUserAllowed()
#4 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter()
#5 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CAccessControlFilter->filter()
#6 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): UserAdminController->filterAccessControl()
#7 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#9 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): UserAdminController->runActionWithFilters()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): UserAdminController->run()
#11 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=userAdmin/admin
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 17:17:04 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 17:20:53 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "userAdmin". in C:\xampp\htdocs\yii-2\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CController.php(270): CController->missingAction('userAdmin')
#1 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('userAdmin')
#2 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('site/userAdmin')
#3 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii-2/index.php?r=site/userAdmin
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 17:21:19 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "userAdmin". in C:\xampp\htdocs\yii-2\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CController.php(270): CController->missingAction('userAdmin')
#1 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('userAdmin')
#2 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('site/userAdmin')
#3 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii-2/index.php?r=site/userAdmin
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 17:22:13 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 17:24:49 [error] [exception.CException] CException: CWebUser and its behaviors do not have a method or closure named "getLevel". in C:\xampp\htdocs\yii-2\framework\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\yii-2\protected\controllers\SiteController.php(88): CComponent->__call('getLevel', Array)
#1 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('login')
#6 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#7 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#8 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/yii-2/index.php?r=site/login
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 17:44:32 [error] [exception.CHttpException.404] CHttpException: Tidak bisa mengurai request "UserController" in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php
---
2023/09/20 17:44:33 [error] [exception.CHttpException.404] CHttpException: Tidak bisa mengurai request "UserController" in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php
---
2023/09/20 17:44:33 [error] [exception.CHttpException.404] CHttpException: Tidak bisa mengurai request "UserController" in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php
---
2023/09/20 17:44:50 [error] [exception.CHttpException.404] CHttpException: Tidak bisa mengurai request "UserAdminController" in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php
---
2023/09/20 17:44:51 [error] [exception.CHttpException.404] CHttpException: Tidak bisa mengurai request "UserAdminController" in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php
---
2023/09/20 17:44:51 [error] [exception.CHttpException.404] CHttpException: Tidak bisa mengurai request "UserAdminController" in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php
---
2023/09/20 17:44:51 [error] [exception.CHttpException.404] CHttpException: Tidak bisa mengurai request "UserAdminController" in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php
---
2023/09/20 17:44:52 [error] [exception.CHttpException.404] CHttpException: Tidak bisa mengurai request "UserAdminController" in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php
---
2023/09/20 17:45:11 [error] [exception.CHttpException.404] CHttpException: Tidak bisa mengurai request "CobaController" in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php
---
2023/09/20 17:45:12 [error] [exception.CHttpException.404] CHttpException: Tidak bisa mengurai request "CobaController" in C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-2/index.php
---
2023/09/20 18:02:32 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 18:02:34 [error] [system.db.CDbCommand] CDbCommand::fetchAll() gagal: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'sik.tbl_user_admin' doesn't exist. Statement SQL yang dijalankan: SHOW FULL COLUMNS FROM `tbl_user_admin`.
in C:\xampp\htdocs\yii-2\protected\views\userAdmin\index.php (19)
in C:\xampp\htdocs\yii-2\protected\controllers\UserAdminController.php (130)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 18:02:34 [error] [exception.CDbException] CDbException: Table "tbl_user_admin" untuk kelas active record "UserAdmin" tidak ditemukan dalam database. in C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php:2391
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(413): CActiveRecordMetaData->__construct(Object(UserAdmin))
#1 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(663): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(1609): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii-2\framework\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#4 C:\xampp\htdocs\yii-2\framework\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#5 C:\xampp\htdocs\yii-2\framework\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#6 C:\xampp\htdocs\yii-2\framework\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#7 C:\xampp\htdocs\yii-2\framework\zii\widgets\CBaseListView.php(125): CDataProvider->getData()
#8 C:\xampp\htdocs\yii-2\framework\zii\widgets\CListView.php(211): CBaseListView->init()
#9 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(147): CListView->init()
#10 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(181): CBaseController->createWidget('zii.widgets.CLi...', Array)
#11 C:\xampp\htdocs\yii-2\protected\views\userAdmin\index.php(19): CBaseController->widget('zii.widgets.CLi...', Array)
#12 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#14 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): CController->renderPartial('index', Array, true)
#16 C:\xampp\htdocs\yii-2\protected\controllers\UserAdminController.php(130): CController->render('index', Array)
#17 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): UserAdminController->actionIndex()
#18 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#19 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#20 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(40): CFilterChain->run()
#21 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#22 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#23 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#24 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#25 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#26 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#27 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#28 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#29 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#30 {main}
REQUEST_URI=/yii-2/index.php
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 18:02:52 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 18:02:54 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 18:36:39 [error] [exception.CException] CException: Properti "User.password2" tidak didefinisikan. in C:\xampp\htdocs\yii-2\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(145): CComponent->__get('password2')
#1 C:\xampp\htdocs\yii-2\framework\validators\CStringValidator.php(79): CActiveRecord->__get('password2')
#2 C:\xampp\htdocs\yii-2\framework\validators\CValidator.php(201): CStringValidator->validateAttribute(Object(User), 'password2')
#3 C:\xampp\htdocs\yii-2\framework\base\CModel.php(159): CValidator->validate(Object(User), Array)
#4 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(810): CModel->validate(NULL)
#5 C:\xampp\htdocs\yii-2\protected\controllers\UserController.php(76): CActiveRecord->save()
#6 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): UserController->actionCreate()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(40): CFilterChain->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#14 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('create')
#16 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('user/create')
#17 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii-2/index.php?r=user/create
HTTP_REFERER=http://localhost/yii-2/index.php?r=user/create
---
2023/09/20 18:37:23 [error] [exception.CException] CException: Properti "User.password2" tidak didefinisikan. in C:\xampp\htdocs\yii-2\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(145): CComponent->__get('password2')
#1 C:\xampp\htdocs\yii-2\framework\validators\CCompareValidator.php(92): CActiveRecord->__get('password2')
#2 C:\xampp\htdocs\yii-2\framework\validators\CValidator.php(201): CCompareValidator->validateAttribute(Object(User), 'password')
#3 C:\xampp\htdocs\yii-2\framework\base\CModel.php(159): CValidator->validate(Object(User), Array)
#4 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(810): CModel->validate(NULL)
#5 C:\xampp\htdocs\yii-2\protected\controllers\UserController.php(76): CActiveRecord->save()
#6 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): UserController->actionCreate()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(40): CFilterChain->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#14 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('create')
#16 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('user/create')
#17 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii-2/index.php?r=user/create
HTTP_REFERER=http://localhost/yii-2/index.php?r=user/create
---
2023/09/20 18:41:00 [error] [system.db.CDbCommand] CDbCommand::fetchAll() gagal: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'sik.tbl_user_admin' doesn't exist. Statement SQL yang dijalankan: SHOW FULL COLUMNS FROM `tbl_user_admin`.
in C:\xampp\htdocs\yii-2\protected\views\userAdmin\index.php (19)
in C:\xampp\htdocs\yii-2\protected\controllers\UserAdminController.php (130)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 18:41:00 [error] [exception.CDbException] CDbException: Table "tbl_user_admin" untuk kelas active record "UserAdmin" tidak ditemukan dalam database. in C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php:2391
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(413): CActiveRecordMetaData->__construct(Object(UserAdmin))
#1 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(663): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii-2\framework\db\ar\CActiveRecord.php(1609): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii-2\framework\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#4 C:\xampp\htdocs\yii-2\framework\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#5 C:\xampp\htdocs\yii-2\framework\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#6 C:\xampp\htdocs\yii-2\framework\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#7 C:\xampp\htdocs\yii-2\framework\zii\widgets\CBaseListView.php(125): CDataProvider->getData()
#8 C:\xampp\htdocs\yii-2\framework\zii\widgets\CListView.php(211): CBaseListView->init()
#9 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(147): CListView->init()
#10 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(181): CBaseController->createWidget('zii.widgets.CLi...', Array)
#11 C:\xampp\htdocs\yii-2\protected\views\userAdmin\index.php(19): CBaseController->widget('zii.widgets.CLi...', Array)
#12 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#14 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): CController->renderPartial('index', Array, true)
#16 C:\xampp\htdocs\yii-2\protected\controllers\UserAdminController.php(130): CController->render('index', Array)
#17 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): UserAdminController->actionIndex()
#18 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#19 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#20 C:\xampp\htdocs\yii-2\framework\web\filters\CFilter.php(40): CFilterChain->run()
#21 C:\xampp\htdocs\yii-2\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#22 C:\xampp\htdocs\yii-2\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#23 C:\xampp\htdocs\yii-2\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#24 C:\xampp\htdocs\yii-2\framework\web\CController.php(291): CFilterChain->run()
#25 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#26 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): CController->run('')
#27 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController('userAdmin')
#28 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#29 C:\xampp\htdocs\yii-2\index.php(13): CApplication->run()
#30 {main}
REQUEST_URI=/yii-2/index.php?r=userAdmin
HTTP_REFERER=http://localhost/yii-2/index.php?r=site/login
---
2023/09/20 18:41:44 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 18:42:35 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 18:42:41 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 18:52:55 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 18:53:06 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 18:53:12 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
2023/09/20 18:55:52 [error] [php] Trying to access array offset on value of type null (C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php:104)
Stack trace:
#0 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-2\framework\web\CBaseController.php(95): AuthitemController->renderInternal()
#2 C:\xampp\htdocs\yii-2\framework\web\CController.php(872): AuthitemController->renderFile()
#3 C:\xampp\htdocs\yii-2\framework\web\CController.php(785): AuthitemController->renderPartial()
#4 C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php(495): AuthitemController->render()
#5 C:\xampp\htdocs\yii-2\framework\web\actions\CInlineAction.php(49): AuthitemController->actionInstall()
#6 C:\xampp\htdocs\yii-2\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii-2\framework\web\CController.php(286): AuthitemController->runAction()
#8 C:\xampp\htdocs\yii-2\framework\web\CController.php(265): AuthitemController->runActionWithFilters()
#9 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(282): AuthitemController->run()
#10 C:\xampp\htdocs\yii-2\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii-2\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii-2\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-2/index.php?r=srbac/authitem/install
in C:\xampp\htdocs\yii-2\protected\modules\srbac\views\authitem\install\install.php (104)
in C:\xampp\htdocs\yii-2\protected\modules\srbac\controllers\AuthitemController.php (495)
in C:\xampp\htdocs\yii-2\index.php (13)
